p2p/transport_test.go

```diff
@@ -569,12 +570,13 @@ func TestTransportHandshake(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	time.Sleep(10 * time.Microsecond)
+	time.Sleep(10 * time.Millisecond)
```

Comment:
why the x1000 change?

Reply:
x1000 is just a random try ;)

Github runner sometimes fails this test, complaining that the port is closed.
Giving it more time helps. Will try with 100 microseconds.

Reply:
100 microseconds seems to work.